### DISCUSSION TITLE
Updated version of fooocus  is a step back

### DISCUSSION THREAD

**Discussion by gutsalovpavel** [lllyasviel/Fooocus#3460] [General]
Updated recently and found that generation quality in the newest version of the fooocus become way worse than in the previous version. It was a masterpiece before, now, it is just a usual stable diffusion engine, but without Loras, control net, so, it is even worse than the usual stable diffusion 
Inpating doesn't work, faces disorder, hands are screwed
The new enhance option makes the quality even lower and just increases all issues 
It is really sad actually, Foocus was very promising, and I advertised it widely among friends, now, I do not want to suggest it to anyone

**Comment by mashb1t** [comment]
Please create an issue and provide a detailed description about what is not working compared to before incl. terminal logs, comparison images, etc.

ps image generation hasn't changed in 2.5.x compared to 2.4.x, only the enhance feature and auto masking has been added.

**Comment by DavidDragonsage** [comment]
My wife and I have used Fooocus a lot since the 2.5 upgrade and not seen any degradation in generation quality or Inpainting quality. In fact, the more we use Fooocus the more we like it - and the more we appreciate it compared to any other AI platform.

The 2.5 upgrade added a lot of new features but actually I have not had time to explore them all. I have been impressed by the ones that I have used. Version 2.5 also solved a long-standing problem I had with unreliable Fooocus startups ("The future belongs to a different loop...").
